lib/poller.js

```diff
@@ -130,13 +131,19 @@ class Poller {
     let allowed = true
     let reason = ''
 
+    if (!role) {
+      return {
+        allowed, reason
+      }
+    }
```

Comment:
might as well merge thisinto the if below

Reply:
yes, that makes sense, thanks!